Form/basics/index.tsx

```diff
@@ -0,0 +1,106 @@
+import React from "react";
```

Comment:
this file is all generic form elements - inputs, checkboxes, etc.

i thought about splitting this file up, but it didn't quite make sense yet. I think if we add like 1 or 2 more elements it will be time to do that. Until then, it's helpful to each element colocated with its style